packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -102,14 +102,45 @@ function create_routes_and_nodes(cwd, config, fallback) {
 		 * @param {import('types').RouteData | null} parent
 		 */
 		const walk = (depth, id, segment, parent) => {
-			if (/\]\[/.test(id)) {
+			const unescaped = id.replace(/\[([ux])\+([^\]]+)\]/i, (match, type, code) => {
```

Comment:
there could be more than one
```suggestion
			const unescaped = id.replace(/\[([ux])\+([^\]]+)\]/gi, (match, type, code) => {
```